addons/comments/initSettings.inc.sqf

```diff
@@ -0,0 +1,70 @@
+[
+    QGVAR(enableComments),
+    "CHECKBOX",
+    [LLSTRING(Comments), LLSTRING(Comments_Description)],
+    [ELSTRING(main,DisplayName), LLSTRING(DisplayName)],
+    true,
+    0,
+    {
+        TRACE_1("Enable comments setting changed",_this);
+
+        private _display = findDisplay IDD_RSCDISPLAYCURATOR;
+        if (!_this || GVAR(enable3DENComments) || isNull _display || GVAR(draw3DAdded)) exitWith {};
+        [_display] call FUNC(addDrawEventHandler);
+    }
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(commentColor),
+    "COLOR",
+    [LLSTRING(CommentColor), LLSTRING(CommentColor_Description)],
+    [ELSTRING(main,DisplayName), LLSTRING(DisplayName)],
+    [1, 1, 0, 0.7],
+    0,
+    {
+        TRACE_1("Comment color setting changed",_this);
+        params ["_r", "_g", "_b", "_a"];
+
+        GVAR(commentsActiveColor) = [_r, _g, _b, 1];
+    }
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(enable3DENComments),
```

Comment:
I think setting could be reworked as "Import 3DEN Comments" and potentially merged with the allow deletion one: "No", "Yes", "Yes + Prevent Deletion".

Reply:
I thought about it and I must say I don't like this. As a client you might want to ignore 3DEN comments but as a server you want to force to preventimg deletion for other clients that see 3DEN comments.